stable/artifactory-ha/values.yaml

```diff
@@ -851,6 +851,7 @@ artifactory:
   ## Customising their resources and java parameters is done here.
   primary:
     name: artifactory-ha-primary
+    preStartCommand:
```

Comment:
Please add a comment that notes this is run in addition (and after) to the main `artifactory.preStartCommand `.

Reply:
@eldada Done